lua/simple-note.lua

```diff
@@ -34,6 +36,7 @@ M.listNotes = function()
       map({ "i", "n" }, "<C-n>", function(prompt_bufnr)
         local full_path = M.createNoteFile()
         local picker = actions_state.get_current_picker(prompt_bufnr)
+
         vim.notify(full_path .. " has been created")
         picker:refresh(finders.new_oneshot_job(find_command, { cwd = M.config.notes_dir }), { reset_prompt = false })
```

Comment:
```suggestion
if full_path ~= nil then
        local picker = actions_state.get_current_picker(prompt_bufnr)

        vim.notify(full_path .. " has been created")
        picker:refresh(finders.new_oneshot_job(find_command, { cwd = M.config.notes_dir }), { reset_prompt = false})
        end
```

or we can move the creation success message to the `createNoteFile` fn

Reply:
Good suggestion, I've moved it towards the `createNoteFile` instead.